Plotter.cpp

```diff
@@ -362,7 +374,7 @@ SResult<> Cut(CutParams p)
 			return Err(std::string("Moving, please try again."));
 		if (resp == "2\x03")
 			return Err(std::string("Empty tray, please load media.")); // Silhouette Cameo
-		return Err("Unexpected response from plotter: '" + resp + "' (" + string_to_hex(resp) + ")");
+		return Err("1 Unexpected response from plotter: '" + resp + "' (" + string_to_hex(resp) + ") instead "+ string_to_hex("1\x03"));
```

Comment:
Ok this is weird, since I don't think you really changed anything before this, but on my CC200-20 on your branch I hit this error with

```
Error: 1 Unexpected response from plotter: '    0,    0\x03' (20202020302c202020203003) instead 3103
```

But on `develop` I don't. Weird right? I'll try to bisect it.

I think I'll also cherry-pick some of your individual commits straight to `develop`. Thanks again for the code!

Reply:
Ah wait, it's nothing to do with your changes actually. It does it on the second cut. Now that I think about it the cutter sounds different after you've cut so there must be some part of the end of cut sequence we're missing or something.

Maybe need to break out wireshark again...

Reply:
In fact, I found that this step depends what is the last usb request before this one. If there was a request and you did not fully read the usb  buffer, then it is not flushed and you have to read more than expected.

I added prior to this call a while loop to read and flush the usb buffer content. Once  you do that, the regmarks only send two response. The first one is always 0, the second one is 1 when it succeed.

I also noticed that the regmarks findiong is using the speed setting of the selected material. The fisrst one in the list (the default one) is really slow.

This is a non sense, registration marks should be done at the highest possible speed. That 's why I also moved the default material to the fifth one, so that it is always fast. This should improve (like using Setting 5, then, once regmarks are found, move back to the user setting) but it can explain why you have different sound, you may have choose a different material there.